### DISCUSSION TITLE
Rinnovo SSL + "Il sito non può fornire una connessione protetta"

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#4003] [Q&A] [answered]
Buongiorno,
devo rinnovare il certificato SSL che scade tra 5 giorni.
Ho fatto un test sul portale non in produzione e da Rete > HTTPS ho fatto disabilita al fine di caricare il nuovo certificato.
Ho provato a caricarlo 2 volte ma evidentemente non ho selezionato quello corretto. Appena ho ricaricato la pagina il sito mi ha chiuso fuori ed il browser risponde con "Il sito non può fornire una connessione protetta".
La prima domanda è: come faccio a tornare in possesso del backend del sito? 
La seconda è: quale è la procedura corretta per eseguire il rinnovo del certificato SSL?
Grazie

**Comment by evilaliv3** [comment]
La procedura che avete seguito è corretta ma evidentemente qualcosa è andato storto ma da ciò che riportate non lo identifico.

Provate ad accedere da un altro browser , o via Tor o via localhost e vedrete che uno di questi modi vi funzionerà

**Reply by maelaborazioni** [reply, marked as answer]
Grazie per i suggerimenti ma purtroppo non ha sortito l'effetto desiderato.
Sono riuscito però a trovare la soluzione che mi ha permesso di riaccedere al backend e ricaricare la private key ed il certificato e quindi ripristinare la connessione al sito.
Riporto il link alla soluzione trovata nel caso qualcuno si imbatta nello stesso problema: https://superuser.com/a/881431
